nav2_voxel_grid/README.md

```diff
@@ -0,0 +1,9 @@
+# Nav2 Voxel Grid
```

Comment:
Non gating question - should Voxel Grid be moved into nav2_utils? I haven't looked at the code, does it need to be it's own package?

Reply:
Potentially, yeah given what `nav2_utils` is right now. I think there's another ticket open for removing `nav2_utils` in favor of something more structured, so I wouldn't add new things to it that already have a convenient break. 
I'm not opposed to it.